trusted-committer/de/04-uplevelling-community-members.asciidoc

```diff
@@ -0,0 +1,23 @@
+
+[[upleveling]]
+== Weiterbildung von Community Mitgliedern
+
+Es gibt ein Kontinuum bei der Teilnahme an einer InnerSource Community. Es gibt Menschen, denen die Community nicht bekannt ist. Neulinge könnten sich für die Community und deren Produkt interessieren, haben aber bisher noch nicht damit gearbeitet. _Consumer_ nutzen die Software, haben aber bisher noch nicht selbst dazu beigetragen.
+Dann gibt es die https://innersourcecommons.org/resources/learningpath/contributor/index[_Contributors_], die schon mindestens einen Beitrag geleistet haben, und schließlich die _Trusted Committer_, welche Verantwortung sowohl für die Software als auch die Community übernehmen.
+Als ein _Trusted Committer_ bist du verantwortlich, Individuen innerhalb dieses Kontinuums zu bewegen und deren Fähigkeiten zu verbessern, Beiträge zu liefern.
```

Comment:
`Individuen innerhalb dieses Kontinuums zu bewegen` sounds a bit like handling of containers at the docks / a bit unpersonal / like moving objects instead of interacting with subjects.

I don't really have a good suggestion for an alternative phrasing right now though.

Reply:
```suggestion
Als _Trusted Committer_ bist du verantwortlich, Individuen zu helfen sich innerhalb dieses Kontinuums weiterzuentwickeln und ihre Fähigkeiten zu verbessern, Beiträge zu liefern.
```